tchannel/request.py

```diff
@@ -0,0 +1,11 @@
+from __future__ import (
+    absolute_import, division, print_function, unicode_literals
+)
+
+
+class Request(object):
+
+    def __init__(self, body=None, headers=None, transport=None):
+        self.body = body
```

Comment:
docs

Use `__slots__` or `namedtuple`. We expect lots of these instances to be created. Should avoid cost of dict.